conda_build/config.py

```diff
@@ -805,5 +805,22 @@ def get_or_merge_config(config, variant=None, **kwargs):
     return config
 
 
+def get_channel_urls(args):
```

Comment:
I am not so confident about this function signature. Is this okay, @msarahan @jjhelmus ?

Reply:
I think ideally you'd have it be "channels" and you'd call ``ensure_list`` on it to make sure it's a list.  However, given that it's already called so awkwardly with either ``channel`` or ``channels``, the way it is is probably the best you can do.  I wouldn't change this right now.  Another wart for the pile.